src/foam/core/Interface.js

```diff
@@ -107,21 +113,37 @@ foam.CLASS({
           ]
         });
 
-        if ( this.client )
-          foam.CLASS({
-            package: this.package,
-            name: 'Client' + this.name,
-            implements: [ this.id ],
-            flags: this.flags,
-            source: this.source,
-            properties: [
-              {
-                class: 'Stub',
-                of: this.id,
-                name: 'delegate'
-              }
-            ]
-          });
+      if ( this.client )
```

Comment:
the first `if` here is a whitespace change - it was inaccurately indented before